src/locales/ko/mystery-encounters/training-session-dialogue.json

```
@@ -1,24 +1,24 @@
 {
   "intro": "훈련에 쓸 만한 물건들을 발견했다.",
   "title": "트레이닝 시간",
-  "description": "발견한 기구들은 포켓몬들을 훈련시키는 데 쓸 수 있을 것 같습니다!\n파티의 다른 포켓몬들과 싸우면서 원하는 방식으로 훈련해 보면 어떨까요?",
+  "description": "발견한 기구들은 포켓몬들을 훈련시키는 데 쓸 수 있을 것 같습니다!\n@[TOOLTIP_TITLE]{1마리를 상대로, 나머지 포켓몬과 함께} 배틀로 훈련해 보면 어떨까요?",
```

```suggestion
  "description": "발견한 기구들은 포켓몬들을 훈련시키는 데 쓸 수 있을 것 같습니다!\n@[TOOLTIP_TITLE]{당신의 포켓몬 1마리를 상대로 하여, 나머지 포켓몬으로 배틀하고 쓰러뜨리며} 훈련해 보면 어떨까요?",
```
부족하게나마 조금 더 의미를 확실하게 해 보았습니다. 
이건 확실히 바뀌는 게 좋기는 하네요. 저는 이거 테스트할 때 고른 포켓몬으로 나머지 포켓몬과 싸우는 건 줄 알았거든요..

---

> 부족하게나마 조금 더 의미를 확실하게 해 보았습니다. 이건 확실히 바뀌는 게 좋기는 하네요. 저는 이거 테스트할 때 고른 포켓몬으로 나머지 포켓몬과 싸우는 건 줄 알았거든요..

그죠? 전 영어 원문을 보고 플레이해서ㅋㅋㅋ
게다가 한국어 스크립트가 제가 번역한 초안이랑 좀 달라졌더라구요. 
어짜피 description은 길이가 상관없으니 반영해두겠습니다!